src/component/Chat/ChatArea/ChatArea.js

```diff
@@ -3,44 +3,112 @@ import React, { useEffect, useState } from "react";
 import "./ChatArea.css"; // Import the CSS file
 import axios from "axios";
 
-const API = axios.create({ baseURL: `http://localhost:4500/` });
+const ENDPOINT = process.env.REACT_APP_ENDPOINT;
+const API = axios.create({ baseURL: ENDPOINT });
 
-const ChatArea = () => {
+const ChatArea = ({ receiverId, senderId, socket }) => {
   const [messageToSend, setMessageToSend] = useState("");
   const [messages, setMessages] = useState([]);
+  const [isSend, setIsSend] = useState();
+  const [conversationId, setConversationId] = useState();
```

Comment:
@athar-shayan why managing state for conversationId, In React state is maintained when updating state would require UI reload. I don't see any need for it

Reply:
Because conversationId is being pass for message sorting in DB.